lib/Db/ColumnTypes/IColumnTypeQB.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace OCA\Tables\DB\ColumnTypes;
+
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use OCP\DB\QueryBuilder\IQueryFunction;
+
+interface IColumnTypeQB {
+	public const DB_PLATFORM_MYSQL = 0;
+	public const DB_PLATFORM_PGSQL = 1;
+	public const DB_PLATFORM_SQLITE = 2;
```

Comment:
Please add another constant for Oracle DB, usually called `oci` in our code. 

Currently, because MySQL is the else in `formatCellValue`, it would use the MySQL dialect, which is not going to work :disappointed: